### PR TITLE
Tool for converting staged contracts migration report from JSON to Markdown

### PR DESCRIPTION
**Pull request by SupunS**
Work towards https://github.com/onflow/cadence/issues/3116

## Description

Converts the json report to a more human readable Markdown report.
Example: https://gist.github.com/SupunS/97bde0154b502c8601087635042bf95d

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Comment by turbolent** [comment]
Nice!

Given the report is generated in flow-go, it might make sense to move the tool there, but given it's temporary, also totally fine to keep it here

File: tools/staged-contracts-report-printer/main.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+			fmt.Sprintf(
+				"| %s | %s | %s | \n",
+				entry.AccountAddress,
+				entry.ContractName,
+				status,
+			),
+		)
+	}
+
+	mdOutput := fmt.Sprintf("%smd", reportPath[0:len(reportPath)-4])
```
Comment (turbolent) [inline tools/staged-contracts-report-printer/main.go:82]: Maybe use `path.Ext` here, e.g. https://www.tumblr.com/golang-examples/118334591809/replace-file-extension

File: tools/staged-contracts-report-printer/main.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	flag.Parse()
+
+	reportPath := *reportPathFlag
+
+	content, err := os.ReadFile(reportPath)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(reportPath)
```
Comment (turbolent) [inline tools/staged-contracts-report-printer/main.go:44]: Debug line?

**Review by turbolent — APPROVED** [review]
LGTM! 👏  Happy to get it merged so we can start using it

File: tools/staged-contracts-report-printer/main.go (hunk 2 above)
Comment (SupunS) [reply tools/staged-contracts-report-printer/main.go]: ah yes, removed

**Review by turbolent — APPROVED** [review]
👌

**Pull request merged**